### DISCUSSION TITLE
Using fine-tuned model with deep_river

### DISCUSSION THREAD

**Discussion by alpcansoydas** [online-ml/river#1206] [Q&A]
Is it possible to use fine-tuned model with PyTorch and deep_river?

**Comment by MaxHalford** [comment]
Can you please ask this in the [deep-river](https://github.com/online-ml/deep-river) repository? There's different people working on that project, I don't know enough to answer.

**Reply by alpcansoydas** [reply]
Of course but i didn't see discussion tab in the deep-river repo.

**Reply by MaxHalford** [reply]
Then open an issue :)
